test/spec/functional/viewAttach.js

```diff
@@ -0,0 +1,277 @@
+var testSrcPath = '../../source',
+    spyOnBackbone = require('./backboneSpy');
+
+describe('A View being deactivated and activated', function() {
+
+  var env, _, $, View;
+
+  beforeEach(function(done) {
+    require('./clientEnv')().done(function(environment) {
+      env = environment;
+      $ = env.window.$;
+      _ = env.window._;
+      View = env.window.Torso.Views.View;
+      $('body').append('<div class="app"></div>');
+      done();
+    });
+  });
+
+  var setUpView = function() {
```

Comment:
Yea, there should now be three commits in the pull request. If not, then
maybe I have to regenerate the pull request
On Apr 10, 2015 9:06 AM, "mandragorn" <notifications@github.com> wrote:

> In test/spec/functional/viewAttach.js
> <https://github.com/vecnatechnologies/backbone-torso/pull/12#discussion_r28142429>
> :
>
> > +describe('A View being deactivated and activated', function() {
> > +
> > +  var env, _, $, View;
> > +
> > +  beforeEach(function(done) {
> > +    require('./clientEnv')().done(function(environment) {
> > +      env = environment;
> > +      $ = env.window.$;
> > +      _ = env.window._;
> > +      View = env.window.Torso.Views.View;
> > +      $('body').append('<div class="app"></div>');
> > +      done();
> > +    });
> > +  });
> > +
> > +  var setUpView = function() {
>
> did you update the pull request? Thanks, ~Josh
> … <#14ca37018969f335_>
> On 04/09/2015 05:40 PM, Kent Willis wrote: In
> test/spec/functional/viewAttach.js <
> https://github.com/vecnatechnologies/backbone-torso/pull/12#discussion_r28105567>:
> > +describe('A View being deactivated and activated', function() { > + > +
> var env, _, $, View; > + > + beforeEach(function(done) { > +
> require('./clientEnv')().done(function(environment) { > + env =
> environment; > + $ = env.window.$; > + _ = env.window._; > + View =
> env.window.Torso.Views.View; > + $('body').append('<div
> class="app"></div>'); > + done(); > + }); > + }); > + > + var setUpView =
> function() { done — Reply to this email directly or view it on GitHub <
> https://github.com/vecnatechnologies/backbone-torso/pull/12/files#r28105567>.
>
>  -- Josh Young Robotics/Software Engineer jyoung@vecna.com (617) 864-0636
> x4548 (office) www.vecna.com/robotics Cambridge Research Laboratory Vecna
> Technologies, Inc. 36 Cambridge Park Drive Cambridge, MA 02140 Office: (617)
> 864-0636 Fax: (617) 864-0638 http://vecna.com Better Technology, Better
> World (TM) The contents of this message may be privileged and confidential.
> Therefore, if this message has been received in error, please delete it
> without reading it. Your receipt of this message is not intended to waive
> any applicable privilege. Please do not disseminate this message without
> the permission of the author.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vecnatechnologies/backbone-torso/pull/12/files#r28142429>
> .
>